tests/integration/test_notebooks_python.py

```diff
@@ -149,19 +149,14 @@ def test_vw_deep_dive_integration(notebooks, size, expected_values):
 
 
 @pytest.mark.integration
-@pytest.mark.skipif(sys.platform == 'win32', reason="nni not installable on windows")
+@pytest.mark.skipif(sys.platform == "win32", reason="nni not installable on windows")
 def test_nni_tuning_svd(notebooks, tmp):
     notebook_path = notebooks["nni_tuning_svd"]
-    # First check if NNI is running
```

Comment:
Shouldn't we keep this? (in case a previous test has failed)

Reply:
I got an error, that's why I removed it. Also, in the notebook, we start by stoping nni. Do you still think we should keep it?

Reply:
Ok, no problem then.